CDP4JsonFileDal/JsonFileDalUtils.cs

```diff
@@ -41,10 +41,15 @@ internal static class JsonFileDalUtils
         /// </summary>
         private const string ExchangeHeaderRemark = "This is an ECSS-E-TM-10-25 exchange file";
 
+        /// <summary>
+        /// The default copyright text to be included in the exchange header file.
+        /// </summary>
+        public const string DefaultHeaderCopyright = "Copyright 2016 © RHEA.";
+
         /// <summary>
         /// The copyright text to be included in the exchange header file.
         /// </summary>
-        private const string ExchangeHeaderCopyright = "Copyright 2016 © RHEA.";
+        public static string ExchangeHeaderCopyright { get; set; }
```

Comment:
i would not make this static, and why don't we allow the API user to update the complete `ExchangeFileHeader` object?

Reply:
Ok.